xcode/Subconscious/Shared/Components/Common/SubtextTextViewRepresentable.swift

```diff
@@ -173,21 +173,25 @@ struct SubtextTextViewRepresentable: UIViewRepresentable {
         NSTextContentManagerDelegate,
         NSTextLayoutManagerDelegate
     {
-        static let renderer = SubtextAttributedStringRenderer()
         /// Is event happening during updateUIView?
         /// Used to avoid setting properties in events during view updates, as
         /// that would cause feedback cycles where an update triggers an event,
         /// which triggers an update, which triggers an event, etc.
         var isUIViewUpdating: Bool
         var representable: SubtextTextViewRepresentable
         var renderTranscludeBlocks: Bool = false
+        /// Subtext renderer instance
+        var renderer: SubtextAttributedStringRenderer
```

Comment:
Make renderer an instance variable. We can change its body size dynamically in `updateUIView` now.